packages/app/.eslintrc.json

```diff
@@ -1,3 +1,7 @@
 {
-    "extends": "next/core-web-vitals"
+    "extends": "next/core-web-vitals",
+    "rules": {
```

Comment:
Adding a couple rules to ignore here, both personal bias so please share any thoughts:

- Turning off exhaustive dependency warnings from "useEffects". We intentionally don't include exhaustive dependencies (see useEffects for first render).
- Making <a> links to pages a warning instead of an error. The only real place we use this is the breadcrumb to return to the homepage. For some reason, that route does not retrigger getServerSideProps.